src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Generated/IOperationsOperations.cs

```diff
@@ -0,0 +1,47 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
@shahabhijeet can you please point to some example spec on what exacty needs to be done here
I tried searching MIT-no-version in azure-rest-api-spec repository but couldnt find any instance of it